.buildkite/pipelines/pull_request/deploy_cloud.yml

```
@@ -2,6 +2,10 @@ steps:
   - command: .buildkite/scripts/steps/cloud/build_and_deploy.sh
     label: 'Build and Deploy to Cloud'
     agents:
-      queue: n2-2
+      queue: n2-2-spot
     depends_on: build
     timeout_in_minutes: 30
+    retry:
+      automatic:
+        - exit_status: '-1'
+          limit: 1
```

```suggestion
          limit: 3
```

Looks like we're using a limit of 3 in other pipelines.